src/services/company.js

```diff
@@ -189,6 +191,14 @@ class CompanyService {
         }
     }
 
+    async releaseOffers(companyId) {
+        try {
+            await Offer.updateMany({ company : companyId }, { $set: { "isPending" : false } });
+        } catch (err) {
+            console.error(err);
+            throw err;
+        }
+    }
```

Comment:
I get where you are coming from with the name of this method, but I think that there should be a better one. Perhaps not and I'm just being picky...

On another note, you have got the wrong attribute for the owning company: instead of `company` it should be `owner`